<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsStacConnectionDialog</class>
 <widget class="QDialog" name="QgsStacConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STAC Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mGroupBox">
     <property name="title">
      <string>Connection Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QLabel" name="lblReferer">
        <property name="text">
         <string>Referer</string>
        </property>
        <property name="buddy">
         <cstring>mEditReferer</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QGroupBox" name="mAuthGroupBox">
        <property name="title">
         <string>Authentication</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QgsAuthSettingsWidget" name="mAuthSettings" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="mEditName">
        <property name="toolTip">
         <string>Name of the new connection</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="mEditReferer">
        <property name="toolTip">
         <string>Optional custom referer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="mEditUrl">
        <property name="toolTip">
         <string>URL of the connection</string>
        </property>
        <property name="placeholderText">
         <string>https://example.com/stac/</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mEditName</tabstop>
  <tabstop>mEditUrl</tabstop>
  <tabstop>mEditReferer</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsStacConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsStacConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
